src/ArrayUtils.php

```diff
@@ -9,17 +9,18 @@
 
 namespace Zend\Stdlib;
 
+use ScriptFUSION\StaticClass;
 use Traversable;
 use Zend\Stdlib\ArrayUtils\MergeRemoveKey;
 use Zend\Stdlib\ArrayUtils\MergeReplaceKeyInterface;
 
 /**
  * Utility class for testing and manipulation of PHP arrays.
- *
- * Declared abstract, as we have no need for instantiation.
  */
-abstract class ArrayUtils
+final class ArrayUtils
 {
+    use StaticClass;
```

Comment:
@0xPaul you are indeed correct that `static class` conveys intent in a clearer way. What isn't clear is how the contract of the class is changed based on that trait by looking at this class.

A private constructor does convey that immediately, but indeed does not express what the meaning is (although developers usually understand that immediately).